deploy.sh

```diff
@@ -0,0 +1,15 @@
+docker_command="docker-compose --env-file .env.production"
```

Comment:
This file is not for deploying, it's more like testing production build on local machine

Reply:
why is that? im not familiar with dev-ops, please explain 😸

Reply:
You deploy to a server, not local machine

Reply:
This script just start docker-compose on local machine, right?

Reply:
yea and if it's on VPS it can also do it right :v

Reply:
How are you gonna run it on VPS?

Reply:
i was thinking of 
1. `git pull production` (or `git clone` if first time)
2. `setup .env.prod` (skip unless first time)
3. `docker-compose`
But nothing concrete for now, havent got much experience.

Reply:
I'm asking how do you run it on VPS, not what you do

Reply:
Also why do you put docker-compose command into a variable?

Reply:
cause typing the whole thing is tedious and uhh i dont want to forget the argument :v, this can be deleted after we finish deploy :3.

Reply:
Why not just write a function then?

Reply:
what function?